src/graphql/root/mutation/intraledger-payment-send.ts

```diff
@@ -22,30 +22,20 @@ const IntraLedgerPaymentSendMutation = GT.Field({
   args: {
     input: { type: GT.NonNull(IntraLedgerPaymentSendInput) },
   },
-  resolve: async (
-    _,
-    args,
-    { domainAccount, logger }: { domainAccount: Account; logger: Logger },
-  ) => {
+  resolve: async (_, args, { domainAccount, logger }) => {
```

Comment:
Not sure if this was an intentional reversion?

Reply:
yes now the graphql domain is automatically been typed

Reply:
this is the PR that brought those changes: https://github.com/GaloyMoney/galoy/pull/891

Reply:
not sure to understand your point @dolcalmi ?

Reply:
ok got it actually